go/base/static/css/campaigns.less

```diff
@@ -93,8 +93,8 @@
     text-align:right; // cancel, save draft, next step
 
     // offsets the cancel button a bit.
-    a:first-child {
-      margin-right:20px;
+    #reset {
+      margin-left: 20px;
     }
```

Comment:
Wondering if this is going to mess up other pages with elements using `reset` as an id, but sounds unlikely. I know we are already using id's in our css in some places (/o\), but maybe we should try avoiding adding more, and rather add a class to the element or something?